scripts/userstyles.yml

```diff
@@ -605,6 +607,17 @@ userstyles:
       app-link: "https://reddit.com"
       current-maintainers: []
       past-maintainers: [*jayylmao, *rubyowo]
+  roblox:
+    name: Roblox
+    categories: [game, entertainment]
+    icon: roblox
+    color: text
+    readme:
+      usage: |+
+      > [!NOTE]
+      > This theme supports the RoPro, RoSeal, RoGold, Roblox+, and BTRoblox third party extensions alongside the base Roblox website. If you experience an issue with the theme, please check if its related to the Roblox website or a third party plugin and mention such in the issue topic.
+      app-link: "https://roblox.com"
+      current-maintainers: [*thatoneunoriginal]
```

Comment:
```suggestion
      usage: |+
        > [!NOTE]
        > This theme supports the RoPro, RoSeal, RoGold, Roblox+, and BTRoblox third party extensions alongside the base Roblox website. If you experience an issue with the theme, please check if its related to the Roblox website or a third party plugin and mention such in the issue topic.
      app-link: "https://roblox.com"
      current-maintainers: [*thatoneunoriginal]
```

Reply:
@ThatOneUnoriginal i believe this should fix the issue.

Reply:
(Yes, that is the issue :p)